src/incrementer.js

```diff
@@ -67,38 +65,38 @@ var Incrementer = {
                         .after(
                             //set up controls
                             '<span class="ctrl ' +
-                            options.incrementerCtrlClass +
-                            '">' +
-                            '<a href="#" class="inc" title="+' +
-                            options.step +
-                            '" tabindex="-1"></a>' +
-                            '<a href="#" class="dec" title="-' +
-                            options.step +
-                            '" tabindex="-1"></a>' +
-                            '</span>'
+                                options.incrementerCtrlClass +
+                                '">' +
+                                '<a href="#" class="inc" title="+' +
+                                options.step +
+                                '" tabindex="-1"></a>' +
+                                '<a href="#" class="dec" title="-' +
+                                options.step +
+                                '" tabindex="-1"></a>' +
+                                '</span>'
```

Comment:
**chore:** Shouldn't this be converted to a string literal?

Reply:
Thanks, done